cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml

```diff
@@ -117,12 +117,17 @@
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro</artifactId>
-      <version>1.7.6</version>
+      <version>${apache.avro.version}</version>
```

Comment:
Should be using dependency management in the parent pom, meaning no version should be specified here.